openedx/core/djangoapps/content/course_overviews/signals.py

```diff
@@ -112,18 +162,60 @@ def _check_for_pacing_changes(previous_course_overview, updated_course_overview)
 
 
 def _check_for_cert_availability_date_changes(previous_course_overview, updated_course_overview):
```

Comment:
[NIT]  In a perfect world, I would change the name of this function as well, probably to

```
def _check_for_cert_date_changes(previous_course_overview, updated_course_overview):
```

just to make it accurately reflect the changed behavior.

Reply:
A good suggestion! I'll update this and I'm also going to update the `_check_for_course_date_changes`'s name to `_check_for_course_start_date_changes`.